server/athenian/api/controllers/contributors_controller.py

```diff
@@ -7,4 +14,29 @@ async def get_contributors(request: web.Request, id: int) -> web.Response:
     :param id: Numeric identifier of the account.
 
     """
-    return web.Response(status=200)
+    user = await request.user()
+
+    async with request.sdb.connection() as sdb_conn:
+        account_id = await sdb_conn.fetch_val(
+            select([UserAccount.account_id])
+            .where(and_(UserAccount.user_id == user.id, UserAccount.account_id == id)))
+        if account_id is None:
+            err_detail = f"Account {account_id} does not exist or user {user} is not a member."
```

Comment:
I'd need to do a separate query to distinguish a 403 (member not part of account) and 404 (account doesn't exist). I saw that we do the same in other endpoints and I think that it doesn't worth the effort now to distinguish them \cc @dpordomingo